docs/tutorial/part-one/index.md

```diff
@@ -15,7 +15,7 @@ Let's check first that you have everything setup to start creating with Gatsby.
 
 Node.js is a programming tool for running JavaScript on servers and in your computer's terminal. Gatsby is built using Node.js.
 
-Open a terminal window and type `node --version` then `npm --version`.
+Open a <a href="http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/">terminal window</a> and type `node --version` then `npm --version`.
```

Comment:
This is for macs. We should also include instructions for opening a terminal in windows. Searching around this looks good https://www.quora.com/How-do-I-open-terminal-in-windows